_posts/2019-08-08-kubernetes-gated-deployments.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Kubernetes Gated Deployments"
+date: 2019-08-08 09:00:00 -0700
+cover: /assets/images/kubernetes-gated-deployments/cover.jpg
+excerpt: Kubernetes Gated Deployments is a Kubernetes controller that facilitates automatic regression testing and canary analysis on Kubernetes deployments, that can detect and roll back the causal change without developer intervention. It is designed to augment existing deployment processes by analyzing key functionality and performance metrics associated with the application.
+authors:
+  - name: Steven Fu
+    title:
+    url:
+    photo:
+  - name: Satish Ravi
+    title:
+    url:
+    photo:
+  - name: Jacob Brooks
+    title:
+    url:
+    photo:
+  - name: Silas Boyd-Wickizer
+    title: Sr. Director of Engineering
+    url: https://github.com/silasbw
+    photo: https://avatars.githubusercontent.com/silasbw
+---
+
+At GoDaddy, teams frequently perform continuous deployments of their services to
+Kubernetes. When engineers make changes to these services, those changes may
+affect the behavior of the application, resulting in a negative impact on the
+customer experience that unit and integration tests would not catch. For
+example, an engineer might introduce a performance regression that increases
+page load time. In this case, it is important to mitigate the impact on
+customers by promptly removing the change from production. Measuring the impact
+of a change and making a quick decision to roll back can be challenging in an
+environment where there are many deploys each day. To help address this
+challenge, we designed, implemented, and open sourced an extension for
+Kubernetes, called Kubernetes Gated Deployments, that automates regression
```

Comment:
This describes the outcome, but doesn't go into detail on how you arrived here. Did you evaluate industry alternatives? What metrics did you care about when making those evaluations, and how did those different tools measure up? 

tl;dr Why should someone consider your solution vs other well-known continuous delivery tools like Spinnaker?